### PR TITLE
modify comments of func IsReady

### LINKED ISSUE #297: Comments modify
I have received you suggestion about my pull request
I have revised the comments of func IsReady of node.go under the path : fluid/pkg/utils/kubeclient  pushed again

### PR DESCRIPTION
**Pull request by hhhercules**
modify the comments of func **IsReady** under the path : **pkg/utils/kubeclient/node.go**

fixes #297

### CODE REVIEW

File: pkg/utils/kubeclient/node.go [hunk 1]
```diff
@@ -38,7 +38,8 @@ func GetNode(client client.Client, name string) (node *v1.Node, err error) {
 	return node, err
 }
 
-// Check if the node is ready
+// IsReady checks if the node is ready
+// If the node is ready,it returns bool type 1,else,it returns bool type 0.
```
Comment (cheyang) [inline pkg/utils/kubeclient/node.go:42]: If the node is ready, it returns true. Otherwise, it returns false.

**Review by cheyang — CHANGES_REQUESTED** [review]

**Comment by cheyang** [comment]
Please open an issue related to it. Thanks.

File: pkg/utils/kubeclient/node.go (hunk 1 above)
Comment (hhhercules) [reply pkg/utils/kubeclient/node.go]: I have added new commits ,could you receive it?

**Comment by cheyang** [comment]
Thank you!

**Review by cheyang — APPROVED** [review]

**Pull request merged**